### PR TITLE
Remove jQuery

### PR DESCRIPTION
**Pull request by SheepFromHeaven**
- No more jquery dependency
- Updated Resource loader
- upgrade @biotope/build to v5.5

### CODE REVIEW

File: src/resources/ts/main.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import ResourceLoader from '@biotope/resource-loader/lib/index.esm';
+
+((window: Window, document: Document) => {
```
Comment (jurekbarth) [inline src/resources/ts/main.ts]: can you change that to plain es6?

File: src/resources/ts/helper.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-(function () {
+(() => {
```
Comment (jurekbarth) [inline src/resources/ts/helper.ts]: can you change that to plain es6?

File: src/resources/ts/main.ts [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+            match: (options) => options.resource.path.indexOf('.js') > -1,
+            handle: (options) => {
+                const script = document.createElement('script');
+                script.src = options.resource.path;
+                script.async = true;
+                document.body.appendChild(script);
+            }
+        }
+
+        const resourceLoader = new ResourceLoader({
```
[the remark this answers is not in the document]
Comment (SheepFromHeaven) [reply src/resources/ts/main.ts]: I already felt bad putting this in the boilerplate anyways. So for my point i feel got this way as it was before.

**Comment by jurekbarth** [comment]
@SheepFromHeaven we're missing polyfills for IE11, anyway we could just wait until the bioElement version is merged, so this one will be obsolete 👍

**Comment by jurekbarth** [comment]
Closing in favor of #79

**Pull request closed without merging**